concepts/sepsis3/abx-poe-list.sql

```diff
@@ -0,0 +1,175 @@
+      when lower(drug) like '%' || lower('septra') || '%' then 1
+      when lower(drug) like '%' || lower('solodyn') || '%' then 1
+      when lower(drug) like '%' || lower('spectracef') || '%' then 1
+      when lower(drug) like '%' || lower('streptomycin sulfate') || '%' then 1
+      when lower(drug) like '%' || lower('sulfadiazine') || '%' then 1
+      when lower(drug) like '%' || lower('sulfamethoxazole') || '%' then 1
+      when lower(drug) like '%' || lower('trimethoprim') || '%' then 1
+      when lower(drug) like '%' || lower('sulfatrim') || '%' then 1
+      when lower(drug) like '%' || lower('sulfisoxazole') || '%' then 1
+      when lower(drug) like '%' || lower('suprax') || '%' then 1
+      when lower(drug) like '%' || lower('synercid') || '%' then 1
+      when lower(drug) like '%' || lower('tazicef') || '%' then 1
+      when lower(drug) like '%' || lower('tetracycline') || '%' then 1
+      when lower(drug) like '%' || lower('timentin') || '%' then 1
+      when lower(drug) like '%' || lower('tobi') || '%' then 1
+      when lower(drug) like '%' || lower('tobramycin') || '%' then 1
+      when lower(drug) like '%' || lower('trimethoprim') || '%' then 1
+      when lower(drug) like '%' || lower('unasyn') || '%' then 1
+      when lower(drug) like '%' || lower('vancocin') || '%' then 1
+      when lower(drug) like '%' || lower('vancomycin') || '%' then 1
+      when lower(drug) like '%' || lower('vantin') || '%' then 1
+      when lower(drug) like '%' || lower('vibativ') || '%' then 1
+      when lower(drug) like '%' || lower('vibra-tabs') || '%' then 1
+      when lower(drug) like '%' || lower('vibramycin') || '%' then 1
+      when lower(drug) like '%' || lower('zinacef') || '%' then 1
+      when lower(drug) like '%' || lower('zithromax') || '%' then 1
+      when lower(drug) like '%' || lower('zmax') || '%' then 1
+      when lower(drug) like '%' || lower('zosyn') || '%' then 1
+      when lower(drug) like '%' || lower('zyvox') || '%' then 1
+    else 0
+    end as antibiotic
+  from `physionet-data.mimic_hosp.prescriptions`
+  where drug_type in ('MAIN','ADDITIVE')
+  -- we exclude routes via the eye, ears, or topically
+  and route not in ('OU','OS','OD','AU','AS','AD', 'TP')
+  and lower(route) not like '%ear%'
+  and lower(route) not like '%eye%'
+  -- we exclude certain types of antibiotics: topical creams, gels, desens, etc
+  and lower(drug) not like '%cream%'
+  and lower(drug) not like '%desensitization%'
+  and lower(drug) not like '%ophth oint%'
+  and lower(drug) not like '%gel%'
+  -- other routes not sure about...
+  -- for sure keep: ('IV','PO','PO/NG','ORAL', 'IV DRIP', 'IV BOLUS')
+  -- ? VT, PB, PR, PL, NS, NG, NEB, NAS, LOCK, J TUBE, IVT
+  -- ? IT, IRR, IP, IO, INHALATION, IN, IM
+  -- ? IJ, IH, G TUBE, DIALYS
+  -- ?? enemas??
+)
+select
+  drug 
+  , count(*) as numobs
+from t1
+where antibiotic = 1
+group by drug
+order by numobs desc;
```

Comment:
order by statements are bad in BigQuery so try to delete all of them that you see.. it's a bad habit of mine and I leave them in here all the time

Reply:
Done